src/main/perl/Fetch/ProfileCache.pm

```diff
@@ -52,7 +52,7 @@ use Digest::MD5 qw(md5_hex);
 use Readonly;
 use LC::Exception qw(SUCCESS);
 use XML::Parser;
-use JSON::XS v2.3.0 qw(decode_json encode_json);
+use JSON::XS 2.3.0 qw(decode_json encode_json);
```

Comment:
Good catch! Clearly adding the `v` in the `repoquery` requirement on SL5 has the effect of matching version `1.43`...